src/app/complex/complex-results/complex-navigator/table-structure/table-structure.component.ts

```diff
@@ -154,12 +165,30 @@ export class TableStructureComponent implements AfterViewInit {
     source.scrolling = true;
     target.scrollTo({left: source.scrollLeft, behavior: 'instant'});
 
-    this.setHorizontalShadowVisibility(source);
+    this.setHorizontalShadowVisibility();
+  }
+
+  @HostListener('window:resize', ['$event'])
```

Comment:
We were missing the resize for this. If you resize the window and then horizontal scrolls is needed, the shadow was not appearing.